torchvision/datasets/voc.py

```diff
@@ -218,6 +218,9 @@ def parse_voc_xml(self, node):
             for dc in map(self.parse_voc_xml, children):
                 for ind, v in dc.items():
                     def_dic[ind].append(v)
+            if (node.tag == 'annotation'):
```

Comment:
nit: can you remove the parenthesis in the `if` condition? It's not needed in Python, and doesn't follow the code-style that we have in the rest of the code.

Also, see https://stackoverflow.com/questions/30549554/python-coding-style-parentheses-in-ifs-loops-etc